### ISSUE TITLE
Code block text color contrast is not accessible

### ISSUE THREAD

**Issue by ronan-hello** [piwik/developer-documentation#800]
Hello,

The color of the text and background of the code block in the plugin documentation is not readable.
You could change the color to white, or add a module like [highlight.js](https://highlightjs.org/) in order to have colorization at the code level.

View the issue: https://plugins.matomo.org/WeatherReports#documentation

Ronan
